app/assets/stylesheets/components/_card.scss

```diff
@@ -0,0 +1,30 @@
+.custom-card {
```

Comment:
Not your fault, but Miho is using the same name for some css classes. Maybe you two already talked about it.